org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaRequestHandler.java

```diff
@@ -34,6 +36,7 @@
   private static final int CHECK_THREAD_INTERVAL_MS = 500;
   private static final int TIMEOUT_WARNING_MS = 20000;
   private static final int TIMEOUT_CLEANUP_BUFFER_MS = 5000;
+  private static String LAMBDA_ID = "";
```

Comment:
nit: initialize to `null` and check if null before assigning?

Reply:
per the [docs](https://docs.aws.amazon.com/lambda/latest/dg/java-handler.html)(search for "initialization code" on that page), the way to docs recommend initializing a static variable would be:
```
private static String LAMBDA_ID = UUID.randomUUID().toString();
```
This way works too, but doing it above the handler is a bit cleaner.

Reply:
looks like alternatively we could create a constructor and initialize things there as well.

Reply:
hah well that is probably where jessie remembered this from :). Simplified the logic, thanks for the info